hamilton/graph.py

```diff
@@ -211,23 +219,22 @@ def display_all(
         :param graphviz_kwargs: kwargs to be passed to the graphviz graph object to configure it.
             e.g. dict(graph_attr={'ratio': '1'}) will set the aspect ratio to be equal of the produced image.
         """
-        defined_nodes = set()
-        user_nodes = set()
+        all_nodes = set()
+        node_modifiers = {}
         for n in self.nodes.values():
             if n.user_defined:
-                user_nodes.add(n)
-            else:
-                defined_nodes.add(n)
+                node_modifiers[n.name] = {"is_user_input": True}
```

Comment:
constants would be nice for these as we use the same value twice. An object/dataclass would work but I think this is overall pretty clean.

Reply:
created enum.